lib/python/picongpu/picmi/diagnostics/checkpoint.py

```diff
@@ -24,62 +24,42 @@ class Checkpoint:
 
     Parameters
     ----------
-    period: TimeStepSpec, optional
-        Specify on which time steps to create checkpoints.
-        Unit: steps (simulation time steps). Required if timePeriod is not provided.
-
+    period: int or TimeStepSpec, optional
+        Number of simulation steps between consecutive checkpoints (e.g., 10 for every 10 steps).
+        Use 0 to disable checkpointing.
+        Alternatively, a TimeStepSpec can be provided for specific step selection
+        (e.g., TimeStepSpec([5, 10]), TimeStepSpec([slice(-10, None, 1)])).
+        Unit: steps or seconds (via TimeStepSpec unit).
     timePeriod: int, optional
-        Specify the interval in minutes for creating checkpoints.
-        Unit: minutes (must be a non-negative integer). Required if period is not provided.
-
+        Time interval between checkpoints in simulation time steps.
+        Use 0 or None to disable time-based checkpointing.
     directory: str, optional
-        Directory inside simOutput for writing checkpoints (default: "checkpoints").
-
+        Directory to store checkpoint files. Default: "checkpoints".
```

Comment:
Why did you remove the useful information, that these directories are created in the simOutput directory?